.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,29 @@
-<!--We’d like to make it as easy as possible for you to contribute to the Docker documentation repository. Before you submit the pull request, we recommend that you review the [Contribution guidelines](/contribute/overview.md). Remove these comments as you go.-->
+<!--
+  Thank you for contributing to Docker documentation!
+
+  Here are a few things to keep in mind:
+
+  - Links between pages should use a relative path
+  - Remember to add an alt text for images
+  - Use a friendly, conversational tone
```

Comment:
I think this line is risky....Might lead to a rush of 'We' '!' etc..

Reply:
Yeh, that could be better. But perhaps we don't need that line at all. Maybe I'm overthinking it though

Reply:
OK, I'll just remove it for now since we are unsure. We can add something back later :-)